.github/workflows/rust.yml

```diff
@@ -82,7 +82,16 @@ jobs:
       - uses: actions-rs/cargo@v1
         with:
           command: test
-          args: -v
+          args: -v  
+      - name: Run cargo-tarpaulin
+        uses: actions-rs/tarpaulin@v0.1
+        with:
+          version: '0.12.4'
+          args: '-- --test-threads 1'
```

Comment:
Why do we need to specify this?

Reply:
Tried without and it seems to be working fine so removed.